app/assets/stylesheets/global.sass

```diff
@@ -1730,47 +1789,34 @@ form
       display: none
     label
       margin-top: 20px
-      font-family: $font-family-sans-serif
-      text-transform: uppercase
-      font-weight: 400
-      font-style: italic
+      font-family: "Merriweather Sans"
+      font-weight: 300 !important
       pointer-events: auto !important
-      border-color: $color-blue
-      color: $color-blue
+      color: #4D4D4D !important
+      top: 0 !important
       span, a
-        font-size: 16px
-        line-height: 24px
+        font-size: 14px
+        line-height: 18px
         vertical-align: middle
-      span
-        color: black
       a
-        color: $color-blue
+        color: #5DA8E6
+        text-decoration: underline
     label:before
+      font-family: 'Material Icons'
       vertical-align: middle
-      content: ''
+      content: '\E835'
```

Comment:
here you can refer to the icons by their textual name, like you do when using the `material-icons` wrapper, so you could change the `content` to just `check_box_outline_blank`